steveDevonport/steve_devonport.md

```diff
@@ -0,0 +1,46 @@
+
+When is a hero not a hero?
+
+In a previous role I was tasked with creating the Test Automation for a number of different web services, however when I joined the team I quickly discovered that they were lacking the capability to deliver customer value rapidly, securely, or with predictability. All this was compounded by the Scrum ceremonies not being orchestrated and there was no Scrum Master. The technology and tooling side was no better.
+
+After being convinced by the recruitment team and interview process that I would be working with an experienced and high performant group of people the above scenario came as a complete shock, whilst there might be a high performing group of people somewhere in the company...I guess I drew the short straw.
```

Comment:
A suggestion would be to stop after complete shock and remove the sentence 'whilst there might be a high performing group of people somewhere in the company...I guess I drew the short straw.'   I don't think it adds much to your story.

Reply:
Thanks.